hydrosdk/data/types.py

```diff
@@ -197,7 +200,30 @@ def proto2np_shape(tsp):
         shape = tuple([int(s.size) for s in tsp.dim])
     return shape
 
+
 # TODO: method not used
 def np2proto_shape(np_shape):
     shape = TensorShapeProto(dim=[TensorShapeProto.Dim(size=x) for x in np_shape])
     return shape
+
+
+def find_in_list_by_name(some_list: list, name: str):
```

Comment:
I supposed using lambda predicate instead of hard coding `.name` accessor, but it'll work for now.